### PR TITLE
chore: add notes for pod-failure chaos

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <str_ruiling@outlook.com>

close https://github.com/chaos-mesh/chaos-mesh/issues/2648
close https://github.com/chaos-mesh/chaos-mesh/issues/2776

This PR would append several notes when using "Pod Failure" chaos experiement.

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @g1eny0ung @en-jin19 @YangKeao

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -147,3 +147,16 @@ PodChaos 是 Chaos Mesh 中的一种故障类型，通过创建 PodChaos 类型
...
 | duration | string | 指定实验的持续时间 | 无 | 是 | 30s |
+
+## “Pod Failure” 混沌实验的一些注意事项
+
+简言之，这里有几个关于使用 "Pod Failure" 混沌实验的建议：
+
+- 如果你正在运行一个气隙或网络隔离的 Kubernetes 集群，请更换一个可用的 "pause image"。
+- 为容器配置 `livenessProbe` 和 `readinessProbe`。
+
+Pod Failure Chaos Experiment would change the `image` of each container in the target Pod to the "pause image", which is a special image that does not perform any operations. We use `gcr.io/google-containers/pause:latest` as the default image as "pause image", and you could change it to any other image in helm values `controllerManager.podChaos.podFailure.pauseImage`. Pod Failure 混沌实验将会改变目标 Pod 中每个容器的 `image` 为 "pause image"，这是一个特殊的镜像，不会执行任何操作。我们使用 `gcr.io/google-containers/pause:latest` 作为默认的 "pause image"，你可以在 helm values `controllerManager.podChaos.podFailure.pauseImage` 中更改它。
```
Comment (g1eny0ung) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md:158]: Is this English paragraph repeated?

**Review by g1eny0ung — APPROVED** [review]
Rest LGTM.

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md (hunk 1 above)
Comment (STRRL) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md]: fixed PTAL!

**Comment by STRRL** [comment]
> @STRRL: PR needs rebase.

conflicts resolved!

File: docs/simulate-pod-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -141,3 +141,16 @@ The following table describes the fields in the YAML configuration file.
 | containerNames | []string | When you configure `action` to `container-killed`, this configuration is mandatory to specify the target container name for injecting faults. | None | No | ['prometheus'] |
 | gracePeriod | int64 | When you configure `action` to `pod-kill`, this configuration is mandatory to specify the duration before deleting Pod. | 0 | No | 0 |
 | duration | string | Specifies the duration of the experiment. | None | Yes | 30s |
+
+## Some Notes for "Pod Failure" Chaos Experiment
```
Comment (cwen0) [inline docs/simulate-pod-chaos-on-kubernetes.md:145]: @en-jin19 PTAL Thanks!

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**